docs/essentials/backgrounds.mdx

```diff
@@ -127,55 +77,31 @@ You don't need additional configuration to get started. But its properties are f
 
 ## API
 
-<Callout variant="warning" icon="🆕">
-
-  **With the [`globals` API](#globals-api)**
+### Globals
 
-  ### Globals
+This module contributes the following globals to Storybook, under the `backgrounds` namespace:
 
-  This addon contributes the following globals to Storybook, under the `backgrounds` namespace:
-
-  #### `grid`
-
-  Type: `boolean`
+#### `grid`
 
-  Whether the [grid](#grid) is displayed.
+Type: `boolean`
 
-  #### `value`
+Whether the [grid](#grid) is displayed.
 
-  Type: `string`
+#### `value`
 
-  When set, the background color is applied and cannot be changed using the toolbar. Must match the key of one of the [available colors](#options).
+Type: `string`
 
-</Callout>
+When set, the background color is applied and cannot be changed using the toolbar. Must match the key of one of the [available colors](#options).
 
 ### Parameters
 
-This addon contributes the following [parameters](../writing-stories/parameters.mdx) to Storybook, under the `backgrounds` namespace:
-
-#### `default`
-
-Type: `string`
-
-Required: See description
-
-Default background color. Must match the `name` property of one of the available colors defined in the [`values`](#values) (or [`options`](#options)) property.
+This module contributes the following [parameters](../writing-stories/parameters.mdx) to Storybook, under the `backgrounds` namespace:
 
 #### `disable`
```

Comment:
TODO: Check that this is truly `disable` and not `disabled` (as it used to be).

Reply:
Confirmed: https://github.com/storybookjs/storybook/blob/next/code/core/src/backgrounds/types.ts#L24